pack/tic/itm.json

```diff
@@ -1,71 +1,178 @@
+        "cost": 2,
+        "deck_limit": 1,
+        "exceptional": true,
+        "faction_code": "rogue",
+        "illustrator": "Julepe",
+        "name": "Lucky Dice",
+        "pack_code": "itm",
+        "position": 307,
+        "quantity": 1,
+        "skill_agility": 1,
+        "skill_willpower": 1,
+        "slot": "Accessory",
+        "subname": "...Or Are They?",
+        "text": "Exceptional.\n[reaction] When you reveal a non-[curse], non-[auto_fail] chaos token, add 1 [curse] token to the chaos bag: Ignore the just revealed chaos token and reveal another one to resolve. If that token has a [curse] or [auto_fail] symbol, return Lucky Dice to your hand (cannot be ignored/canceled).",
+        "traits": "Item. Relic.",
+        "type_code": "asset",
+        "xp": 3
+    },
+    {
+        "code": "07308",
+        "cost": 0,
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "illustrator": "Adam S. Doyle",
+        "name": "Rite of Equilibrium",
+        "pack_code": "itm",
+        "position": 308,
+        "quantity": 2,
+        "skill_wild": 1,
+        "text": "Either (choose one):\n- Add X [curse] tokens to the chaos bag to add X [bless] tokens to the chaos bag.\n- Remove X [curse] and X [bless] tokens from the chaos bag to heal X total horror from among cards at your location.",
+        "traits": "Spell. Blessed. Cursed.",
+        "type_code": "event",
+        "xp": 5
+    },
+    {
+        "code": "07309",
+        "cost": 3,
+        "deck_limit": 2,
+        "faction_code": "survivor",
+        "illustrator": "Tony Foti",
+        "is_unique": true,
+        "name": "Jacob Morrison",
+        "pack_code": "itm",
+        "position": 309,
+        "quantity": 2,
+        "skill_wild": 1,
+        "slot": "Ally",
+        "subname": "Coast Guard Captain",
+        "text": "Jacob Morrison does not ready during the upkeep phase.\n[reaction] When you would fail a skill test, exhaust Jacob Morrison: You get +2 skill value for that test.\n[reaction] After a [bless] token is revealed from the chaos bag during a skill test you are performing: Ready Jacob Morrison.",
+        "traits": "Ally. Blessed.",
+        "type_code": "asset",
+        "xp": 3
+    },
+    {
+        "code": "07310",
+        "cost": 1,
+        "deck_limit": 2,
+        "faction_code": "survivor",
+        "illustrator": "Alexander Chelyshev",
+        "is_unique": true,
+        "name": "Shrine of the Moirai",
+        "pack_code": "itm",
+        "position": 310,
+        "quantity": 2,
+        "skill_agility": 1,
+        "skill_intellect": 1,
+        "skill_willpower": 1,
+        "subname": "Coast Guard Captain",
+        "text": "Attach to your location. Uses (3 offerings).\nAttached location gains: \"[fast] Draw the top card of the encounter deck, exhaust Shrine of the Moirai, and spend 1 offering: Return up to 2 cards with a total combined level of 5 or less from your discard pile to your hand.\"",
```

Comment:
Please add in bold as a new line at the end of the text field: "Card design by the Moirai at Arkham Nights 2019."

Reply:
Ahh, wasn't sure we did this.  Ikiaq doesn't have this text on her right now.